src/Components/common/Footer.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+const Container = styled.footer`
+  width: 100%;
+  height: 160px;
+  background-color: var(--black-color);
+  margin-top: 60px;
+
+  @media (max-width: 767px) {
+    position: relative;
+    padding: 32px 32px 64px 32px;
+  }
+`;
+
+const Layout = styled.div`
+  max-width: 1400px;
+  padding: 32px 16px 0 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin: 0 auto;
+`;
```

Comment:
@78-artilleryman 

Container라는 이름과 Layout이라는 이름이 조금은 ambiguous 해보입니다.
프론트엔드 진영에서 컨테이너와 레이아웃이라는 이름은 약간 약속된것처럼 의미하는 바가 존재하는데요.
컨테이너는 max-width를 설정해 해당 바운더리를 벗어나지 않도록 하는 말 그대로 container 요소들을 표현하고
레이아웃은 모든 콤포넌트에 공통적으로 활용되어야 하는 스타일링 및 UI 배치를 뜻합니다. 

만약 네이밍이 중복되어 어렵다면 StyledComponent의 규칙을 따라 
스타일성 콤포넌트는 `Styled` 를 앞에 prefix로 붙여 활용해보는건 어떨까요? 

```suggestion
const StyledFooter = styled.footer`
  width: 100%;
  height: 160px;
  background-color: var(--black-color);
  margin-top: 60px;

  @media (max-width: 767px) {
    position: relative;
    padding: 32px 32px 64px 32px;
  }
`;

const StyledContainer = styled.div`
  max-width: 1400px;
  padding: 32px 16px 0 16px;
  display: flex;
  justify-content: space-between;
  align-items: flex-start;
  margin: 0 auto;
`;
```